common.gypi

```diff
@@ -471,6 +471,13 @@
             'ldflags': [
               '-Wl,-blibpath:/usr/lib:/lib:/opt/freeware/lib/pthread/ppc64',
             ],
+            'conditions': [
+              [ 'node_shared=="true"', {
+                'ldflags': [
+                  '-Wl,-bnoipath', # Do not add the full path to libnode.a in executables, trust that the LIBPATH is set correctly by the user at runtime
```

Comment:
I think this needs to have the comment be in lines above the flags so that we can keep it within 80 char limit we have set in the project.

Reply:
Sorry I pushed some work-in-progress AIX changes to this branch instead of a new one! I'll remove the AIX changes from the PR.